Gems/EMotionFX/Code/EMotionFX/Source/AnimGraphGameControllerSettings.h

```diff
@@ -165,6 +163,6 @@ namespace EMotionFX
 
     private:
         AZStd::vector<Preset*>   m_presets;
-        AZ::u32                  m_activePresetIndex;
+        AZ::u64                  m_activePresetIndex;
```

Comment:
What were the rules to change these automatically? Just curious why this got changed to a 64-bit int.

Reply:
This variable represents an index into a vector. Indexes are of type `size_t`. But this variable is also serialized. Largest `size_t` we support is 64 bits, so this variable became a `AZ::u64`.